### ISSUE TITLE
chrome_extensions table doesn't work on OS X 10.10 with Chrome 42

### ISSUE THREAD

**Issue by brandonweeks** [facebook/osquery#1017]
Running latest osquery built from source the chrome_extensions table is not being populated.

Results:
     osquery> SELECT * FROM chrome_extensions;
     osquery> 

Expected results: table would be populated with a number of extensions that are present in Chrome.

It appears the problem is the name of the Chrome Profile path, the path on my Chrome install is
> /Users/brandon.weeks/Library/Application Support/Google/Chrome/Profile 2

https://github.com/facebook/osquery/blob/master/osquery/tables/applications/browser_chrome.cpp hard codes this to "/Library/Application Support/Google/Chrome/Default/"
```
#ifdef __APPLE__
#define kChromePath "/Library/Application Support/Google/Chrome/Default/"
#else
```

From the Chrome source the profiles names are defined in "chrome/common/chrome_constants.h"

```
const char    kInitialProfile[] = "Default";
const char    kMultiProfileDirPrefix[] = "Profile ";
```

And in "chrome/browser/profiles/profile_manager.cc" we see how the names are selected
```
base::FilePath ProfileManager::GenerateNextProfileDirectoryPath() {
  PrefService* local_state = g_browser_process->local_state();
  DCHECK(local_state);

  DCHECK(profiles::IsMultipleProfilesEnabled());

  // Create the next profile in the next available directory slot.
  int next_directory = local_state->GetInteger(prefs::kProfilesNumCreated);
  std::string profile_name = chrome::kMultiProfileDirPrefix;
  profile_name.append(base::IntToString(next_directory));
  base::FilePath new_path = user_data_dir_;
#if defined(OS_WIN)
  new_path = new_path.Append(base::ASCIIToUTF16(profile_name));
#else
  new_path = new_path.Append(profile_name);
#endif
  local_state->SetInteger(prefs::kProfilesNumCreated, ++next_directory);
  return new_path;
}
```

**Comment by brandonweeks** [comment]
It appears the problem is the name of the Chrome Profile path, the path on my Chrome install is
> /Users/brandon.weeks/Library/Application Support/Google/Chrome/Profile 2

as opposed to my coworkers profile path:
> /Users/user/Library/Application Support/Google/Chrome/Default

**Comment by theopolis** [comment]
Excellent, thanks @brandonweeks for doing the investigation into Chrome's source (and profile paths) we should have done initially. :P

**Comment by theopolis** [comment]
Fix: #1054
